components/authz-service/storage/v2/postgres/postgres_test.go

```diff
@@ -19,14 +19,26 @@ import (
 	storage "github.com/chef/automate/components/authz-service/storage/v2"
 	"github.com/chef/automate/components/authz-service/testhelpers"
 	"github.com/chef/automate/lib/grpc/auth_context"
-	uuid "github.com/chef/automate/lib/uuid4"
 )
 
 const (
 	Applied = "applied"
 	Staged  = "staged"
 )
 
+const (
+	policyMembersByMemberName = "SELECT count(*) FROM iam_policy_members WHERE member_id=member_db_id($1)"
+	policyMembersByPolicyID   = "SELECT count(*) FROM iam_policy_members WHERE policy_id=policy_db_id($1)"
+	policyMembersFull         = "SELECT count(*) FROM iam_policy_members WHERE policy_id=policy_db_id($1) and member_id=member_db_id($2)"
+	policyWithID              = "SELECT count(*) FROM iam_policies WHERE id=$1"
+	statementWithID           = "SELECT count(*) FROM iam_statements WHERE db_id=$1"
+	statementQueryWithRole    = "SELECT count(*) FROM iam_statements WHERE resources=$1 AND actions=$2 AND effect=$3 AND policy_id=policy_db_id($4) AND role_id=role_db_id($5)"
+	statementQueryFull        = "SELECT count(*) FROM iam_statements WHERE resources=$1 AND actions=$2 AND effect=$3 AND policy_id=policy_db_id($4)"
+	policyFull                = "SELECT count(*) FROM iam_policies WHERE id=$1 AND name=$2 AND type=$3"
+	membersCount              = "SELECT count(*) FROM iam_members"
+	policyProjectsByProjectID = "SELECT count(*) FROM iam_policy_projects WHERE policy_id=policy_db_id($1)"
```

Comment:
Ok, so this is a nit but I think it would be really helpful if these 
1) Used the plural noun form (given that the `SELECT` is from a pluralized table) and 
2) Used the preposition `with`
3) Drop the word `Query`

So `statementWithID` becomes `statementsWithID` and `policyMembersByPolicyID ` becomes `policyMembersWithPolicyID`.

Mostly I think this makes the expectations easier to read:
`assertOne(t, db.QueryRow(policyFull, polID, name, typeVal.String()))`
vs
`assertOne(t, db.QueryRow(policiesWithIDName, polID, name, typeVal.String()))`

Reply:
Also, barring other blockers/you hating this idea, I'm glad to push up a commit in your absence

Reply:
Yeah please do! I'm afraid I haven't spent a lot of thought there, this was an off-the-cuff thing. 😄

Reply:
I'd rather merge this now 😄 happy to review any small follow-up cleanups.